### PR TITLE
feat: Add toggle command

### LINKED ISSUE #378: Command to toggle coverage
Instead of having two separate keybindings for displaying and removing coverage gutters, it would be great if I could bind a single shortcut to toggle it on or off.

This would be similar to e.g. how GitLens controls blame annotations.

### PR DESCRIPTION
**Pull request by anishkny**
## Issue:
- #378 

## Changes:
- Add new boolean to track current display state.
- Add command to show/hide coverage gutter based on boolean.
- Tests and code linting are passing via continuous integration checks.

### CODE REVIEW

**Comment by anishkny** [comment]
Hmm not sure why this is failing. Seems to pass on my fork: https://github.com/anishkny/vscode-coverage-gutters/actions/runs/3462820220/jobs/5782213247

**Comment by ryanluker** [comment]
> Hmm not sure why this is failing. Seems to pass on my fork: https://github.com/anishkny/vscode-coverage-gutters/actions/runs/3462820220/jobs/5782213247

Thanks for the PR @anishkny !
I think a recently visual studio code insiders change has broken the integration tests I have 🤔.

I will have to investigate when I get some free time next...

**Comment by anishkny** [comment]
> Thanks for the PR @anishkny ! I think a recently visual studio code insiders change has broken the integration tests I have 🤔. This started occuring here in the nightly scheduled builds that we do. https://github.com/ryanluker/vscode-coverage-gutters/actions/runs/3484203088/jobs/5847104491
> 
> I will have to investigate when I get some free time next...

Ok thanks. The error doesn't seem related. I will leave it with you. LMK in case of any feedback.

**Comment by ryanluker** [comment]
@anishkny There was an issue with the insiders vscode testing suite.
I have fixed up the latest master to get around this issue, please update your branch when you can, and I can give it a final review (looking good though!)

**Comment by anishkny** [comment]
@ryanluker Done. Synced my fork and re-ran CI. Seems to pass: https://github.com/anishkny/vscode-coverage-gutters/actions/runs/3509504510

File: package.json
Comment (ryanluker) [inline package.json:167]: You might also want to add a key binding, but I am not sure which you might use 🤔.
https://github.com/ryanluker/vscode-coverage-gutters/pull/385/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L177-L204

File: src/extension/gutters.ts
Comment (ryanluker) [inline src/extension/gutters.ts:86]: Looking pretty good overall, we might want an integration test to confirm that the toggling works how we expect (see link below to the current ones we have in the test suite).

I might also have some free time today to add that test coverage before we merge this 😁.
https://github.com/ryanluker/vscode-coverage-gutters/blob/master/test/extension.test.ts#L184-L205

File: package.json
Comment (anishkny) [reply package.json:167]: Yes I thought about it. How about `ctrl/cmd+shift+5`?

File: src/extension/gutters.ts
Comment (anishkny) [reply src/extension/gutters.ts:86]: Yes. I looked into adding an integration test a little bit but wasn't quite able to figure it out. Be great if you could do the honours - I'll follow along.

File: package.json
Comment (anishkny) [reply package.json:167]: Added this stanza for now, feel free to tweak:

```json
{
  "command": "coverage-gutters.toggleCoverage",
  "key": "ctrl+shift+5",
  "mac": "shift+cmd+5"
}
```

File: src/extension/gutters.ts
Comment (ryanluker) [reply src/extension/gutters.ts:86]: @anishkny I pushed a test for us to use, I also tested out the feature locally and I think it is working well!
There is a bit of oddity with how it interacts with the watch command (it removes the coverage from the current page but doesn't disable the watching... maybe we want it like that?)

File: package.json
Comment (ryanluker) [reply package.json:167]: Looks good to me! I think the only keybinding we can't use is ctrl+shift+0 due to vscode not seeing it for some reason (based on a previous ticket).

File: src/extension/gutters.ts
Comment (ryanluker) [reply src/extension/gutters.ts:86]: Oopps I can't push to your branch in your fork... I pushed the commit here for you to grab when you get a chance.
https://github.com/anishkny/vscode-coverage-gutters/pull/1

Comment (anishkny) [reply src/extension/gutters.ts:86]: Thanks for adding the test! Merged into this PR.

Passing:

https://github.com/anishkny/vscode-coverage-gutters/actions/runs/3511153332/jobs/5882576842#step:6:108

```
✔ Run toggle coverage on python test file x2 @integration (77ms)
```

**Review by ryanluker — APPROVED** [review]
Nice work, thanks for the contribution!

**Comment by anishkny** [comment]
@ryanluker Gentle ping to get this merged. I do not have write access.

**Comment by ryanluker** [comment]
> @ryanluker Gentle ping to get this merged. I do not have write access.

Whoops forgot to hit the green go button hah...

**Pull request merged**
